index.js

```diff
@@ -0,0 +1,126 @@
+require('dotenv').config();
+
+const express = require('express');
+const cors = require('cors');
+const bcrypt = require('bcryptjs');
+const jwt = require('jsonwebtoken');
+const db = require('./database/dbConfig.js');
+const server = express();
+
+server.use(express.json());
+server.use(cors());
+
+// ======== SERVER RUNNING ======== //
+server.get('/', (req, res) => {
+  res.send('<h1>Its Alive!<h1>');
+});
+
+
+// ========= CREATES USER ========= //
+server.post('/api/register', (req, res) => {
+  const credentials = req.body;
+  const hash = bcrypt.hashSync(credentials.password, 10);
+  credentials.password = hash;
+
+  db('users')
+    .insert(credentials)
+    .then(ids => {
+      const id = ids[0];
+      const token = generateToken({ username: credentials.username });
+      res.status(201).json({ 
+        newUserId: id, 
+        token, 
+        message: `Welcome to the database, ${credentials.username}` });
+    })
+    .catch(err => {
+      res.status(500).json(err);
```

Comment:
Write an error message explaining why the user failed to register and how they can fix it.